src/sparse/KokkosSparse_sptrsv_supernode.hpp

```diff
@@ -476,13 +480,19 @@ generate_supernodal_graph(bool col_major, graph_t &graph, int nsuper, const inpu
   // count non-empty supernodal blocks
   row_map_view_host_t hr ("rowmap_view", nsuper+1);
   integer_view_host_t check ("check", nsuper);
+  integer_view_host_t  idxs ("idxs",  nsuper);
   Kokkos::deep_copy (hr, 0);
   Kokkos::deep_copy (check, 0);
 
+  #ifdef KOKKOS_SPTRSV_SUPERNODE_PROFILE
+  timer.reset ();
+  #endif
   int nblocks = 0;
   for (int s = 0; s < nsuper; s++) {
     int j1 = nb[s];
     int j2 = j1+1;  // based on the first row
+
+    size_type nidxs = 0;
     for (size_type i = row_map_host (j1); i < row_map_host (j2); i++) {
       int s2 = map (entries_host (i));
```

Comment:
Is `s2` the index of a 2-rank matrix (supernodal block?) that contains non-zero scalars?